extensions/aris/src/bigDataCluster/controller/apiGenerated.ts

```diff
@@ -0,0 +1,1654 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as request from 'request';
```

Comment:
Do we have a script or documentation in the project indicating how this file was generated?

Reply:
no. Go to http://editor.swagger.io/, and put your yaml or json. It will give you downloadable API.